homeassistant/components/sensor/homematicip.py

```diff
@@ -0,0 +1,418 @@
+"""
+Support for HomematicIP sensors.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/homematicip/
+"""
+
+import logging
+
+from homeassistant.core import callback
+from homeassistant.helpers.entity import Entity
+from homeassistant.components.homematicip import (DOMAIN,
+                                                  EVENT_HOME_CHANGED,
+                                                  EVENT_DEVICE_CHANGED)
+
+REQUIREMENTS = ['homematicip==0.8']
```

Comment:
platform should not have the requirement. Instead, they should have a dependency on the main component.